contracts/truefi/LoanToken.sol

```diff
@@ -303,11 +307,13 @@ contract LoanToken is ILoanToken, ERC20 {
      */
     function redeem(uint256 _amount) external override onlyClosed {
         uint256 amountToReturn = _amount.mul(_balance()).div(totalSupply());
+        uint256 truToReturn = _amount.mul(tru.balanceOf(address(this))).div(totalSupply());
         redeemed = redeemed.add(amountToReturn);
         _burn(msg.sender, _amount);
         require(currencyToken.transfer(msg.sender, amountToReturn));
+        require(tru.transfer(msg.sender, truToReturn));
```

Comment:
Maybe some small optimisation like `if(truToReturn)`? This is gonna be very rare case anyway.